src/sentry/static/sentry/app/views/eventsV2/relatedEvents.tsx

```diff
@@ -70,6 +73,15 @@ class RelatedEvents extends AsyncComponent<Props> {
     return <Placeholder height="120px" bottomGutter={2} />;
   }
 
+  renderError(error) {
+    // Hide the related events if the user doesn't have global-views
+    if (!this.hasGlobalViews()) {
+      return <span />;
```

Comment:
what about `null`?

Reply:
I tried null, but typescript complained about the return types not being consistent with the parent.

Reply:
Does it work if we specify a return type in parent class of `React.ReactNode`?